app/javascript/react/screens/App/Overview/components/ScheduleMigrationModal/ScheduleMigrationModal.js

```diff
@@ -1,11 +1,22 @@
+import moment from 'moment';
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Button, Modal } from 'patternfly-react';
 import ScheduleMigrationModalBody from './ScheduleMigrationModalBody';
 import getPlanScheduleInfo from '../Migrations/helpers/getPlanScheduleInfo';
 
 class ScheduleMigrationModal extends React.Component {
-  state = { dateTimeInput: '' };
+  constructor(props) {
+    super(props);
+
+    const { scheduleMigrationPlan } = this.props;
+    const { migrationScheduled } = getPlanScheduleInfo(scheduleMigrationPlan) || '';
```

Comment:
I think we need to look at the `getPlanScheduleInfo` helper and make sure it is aware of warm migrations, because currently the `migrationScheduled` value here will always be the cold-migration scheduled time. So if the user already scheduled a warm migration cutover and they open the modal again to edit that schedule, the state won't pre-populate with their cutover time.

Reply:
Yes, added `scheduleCutover` to the returned object.